packages/admin-ui/src/Accordion/Accordion.tsx

```diff
@@ -2,22 +2,18 @@ import React from "react";
 import { makeDecoratable, withStaticProps, cva, type VariantProps, cn } from "~/utils.js";
 import type { AccordionRoot } from "./components/AccordionRoot.js";
 import { AccordionItem, type AccordionItemProps } from "./components/AccordionItem.js";
+import { DepthProvider, useDepth } from "./DepthContext.js";
 
 const accordionVariants = cva("wby-group wby-w-full", {
     variants: {
         variant: {
-            container: "wby-accordion-variant-container wby-gap-xs wby-flex wby-flex-col",
-            underline: "wby-accordion-variant-underline "
-        },
-        background: {
-            base: "wby-accordion-background-base",
```

Comment:
Why did we loose bg prop? We still need it per DS: https://www.figma.com/design/f0QUDWX37Kt5X53eltTRiT/Webiny-Design-System?node-id=3253-1490&t=KDI8H2G0VBoKW34F-4